docs_app/content/guide/operators.md

```diff
@@ -296,41 +288,60 @@ function discardOddDoubleEven() {
 
 It is more complicated, but if you have to write an operator that cannot be made from a combination of existing operators (a rare occurrance), you can write an operator from scratch using the Observable constructor, like this:
 
-
 ```ts
-import { Observable } from 'rxjs';
+import { Observable, of } from 'rxjs';
 
 function delay(delayInMillis) {
```

Comment:
Ben, what do you think about introducing types and generics to this example? Many people, like I did, will copy/paste this example and may get TS errors like `TS7006: Parameter 'delayInMillis' implicitly has an 'any' type.` and similar.
These errors may depend on the TS configuration, but I think that no matter the configuration, the `timerID` that is passed to the `clearTimeout` in Observable's teardown function will produce an error: `Type 'unknown' is not assignable to type 'number'.`. This could be fixed by declaring a type over a `Set`: `const allTimerIDs = new Set<number>();`